src/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.Extract.cs

```diff
@@ -0,0 +1,197 @@
+        /// is interpreted as relative to the current working directory. If a file to be archived has an invalid last modified
+        /// time, the first datetime representable in the Zip timestamp format (midnight on January 1, 1980) will be used.
+        /// </summary>
+        /// 
+        /// <exception cref="ArgumentException">sourceArchive or destinationDirectoryName is a zero-length string, contains only whitespace,
+        /// or contains one or more invalid characters as defined by InvalidPathChars.</exception>
+        /// <exception cref="ArgumentNullException">sourceArchive or destinationDirectoryName is null.</exception>
+        /// <exception cref="PathTooLongException">sourceArchive or destinationDirectoryName specifies a path, file name,
+        /// or both exceed the system-defined maximum length. For example, on Windows-based platforms, paths must be less than 248 characters,
+        /// and file names must be less than 260 characters.</exception>
+        /// <exception cref="DirectoryNotFoundException">The path specified by sourceArchive or destinationDirectoryName is invalid,
+        /// (for example, it is on an unmapped drive).</exception>
+        /// <exception cref="IOException">The directory specified by destinationDirectoryName already exists.
+        /// -or- An I/O error has occurred. -or- An archive entry?s name is zero-length, contains only whitespace, or contains one or
+        /// more invalid characters as defined by InvalidPathChars. -or- Extracting an archive entry would result in a file destination that is outside the destination directory (for example, because of parent directory accessors). -or- An archive entry has the same name as an already extracted entry from the same archive.</exception>
+        /// <exception cref="UnauthorizedAccessException">The caller does not have the required permission.</exception>
+        /// <exception cref="NotSupportedException">sourceArchive or destinationDirectoryName is in an invalid format. </exception>
+        /// <exception cref="FileNotFoundException">sourceArchive was not found.</exception>
+        /// <exception cref="InvalidDataException">The archive specified by sourceArchive: Is not a valid ZipArchive
+        /// -or- An archive entry was not found or was corrupt. -or- An archive entry has been compressed using a compression method
+        /// that is not supported.</exception>
+        /// 
+        /// <param name="sourceArchiveFileName">The path to the archive on the file system that is to be extracted.</param>
+        /// <param name="destinationDirectoryName">The path to the directory on the file system. The directory specified must not exist, but the directory that it is contained in must exist.</param>
+        public static void ExtractToDirectory(string sourceArchiveFileName, string destinationDirectoryName) => ExtractToDirectory(sourceArchiveFileName, destinationDirectoryName, entryNameEncoding: null, overwrite: false);
```

Comment:
Nit: These are long lines... maybe you could split them, at least moving this to the next line and indenting.